APIBacklog/documentation/SupportingDocuments/API proposals/APIproposal_BlockchainPublicAddress_Telefonica.md

```diff
@@ -0,0 +1,12 @@
+| **Field** | Description | 
+| ---- | ----- |
+| API family name | Blockchain Public Address |
+| API family owner| Telefonica  |
+| API summary | **High level description**<br> This API allows to manage a Blockchain Public Address associated to a phone number, i.e. to retrieve the blockchain public address(es) and to bind/unbind a Blockchain Public Address. The Block Public Address is referred to as Decentralized IDentifier (DID). With the proposed API, telco service providers have the opportunity to provide 3rd parties with the following capability: pairing phone number with DID whenever this 3rd party wants to offer its customers a way to make transactions more easily based on the phone number instead of the DID. <br><br>**Rationale**<br>DID is composed of 26-35 alphanumeric characters in length. This identifier is not memorisable or usable by a person, so there is the possibility of positioning the telephone number as the predominant identifier on web3. This new identity protocol will make it possible to have a system similar to the Telcos' bizum in web3 ecosystem.
+| Technical viability | For this API to work, the following is needed: <li> Wallets, which will be in charge of generating the DID.</li><li> Storing this DID together with the user's phone number, with prior validation that the targeted user has telephone number. </li><li>Designing this API for it to be interoperable among the different operators</li>The operators will make this API available to 3rd parties through their API marketplaces. The invokation of this API will allow operators to detect possible fraudulent uses. |
```

Comment:
s/invokation/invocation

2)	I do not understand the statement “The invocation of this API will allow operators to detect possible fraudulent uses.” I assume it should say, the operators have to protect the API against fraudulent use. Is there a proposal to define this?

Also how consent of sending AND receiving user will be managed is still to be defined. In general, the proposed mapping may have problems  from a privacy point of view because if I know your wallet address I can see all your transaction done with this address. i.e. The database will be an attractive target of hackers

Reply:
Text is updated according to your suggestion in this commit: https://github.com/camaraproject/WorkingGroups/pull/154/commits/6bec4f9f8a0c0a35295992f9c57c2c4ced689ad5

Regarding consent, yes, it's very important in this API. There is a sub-project for Identity&Consent that should handle these topics and this API has to adhere to conclusions there.